client/my-sites/stats/stats-detail-months/index.jsx

```diff
@@ -128,32 +129,27 @@ export default React.createClass( {
 						</li>
 					</ul>
 				</div>
-				<div className="module-content">
-					<div className="module-content-text module-content-text-info">
-						<p className="message">
-							{ this.translate( 'No views yet', { context: 'Empty info box title for post stats page in Stats' } ) }
-						</p>
-						<p>
-							{ this.translate( 'This panel gives you an overview of how many views your website gets on average.', { context: 'Info box description for post stats page in Stats' } ) }
-						</p>
-						<p className="legend achievement">{
-							this.translate(
-								'%(value)s = The all-time highest value.',
-								{ args:
-									{ value: ( this.numberFormat( highest ) ) },
-									context: 'Legend for post stats page in Stats'
-								}
-							)
-						}</p>
-					</div>
-					<div className="module-placeholder is-void"></div>
-					<div className="module-content-table">
-						<div className="module-content-table-scroll">
-							<table cellPadding="0" cellSpacing="0">
-								{ tableHeader }
-								{ tableBody }
-							</table>
-						</div>
+				<StatsModuleContent className="module-content-text-info">
+					<p>
+						{ this.translate( 'This panel gives you an overview of how many views your website gets on average.', { context: 'Info box description for post stats page in Stats' } ) }
+					</p>
+					<span className="legend achievement">{
+						this.translate(
+							'%(value)s = The all-time highest value.',
+							{ args:
+								{ value: ( this.numberFormat( highest ) ) },
+								context: 'Legend for post stats page in Stats'
+							}
+						)
+					}</span>
+				</StatsModuleContent>
+				<div className="module-placeholder is-void"></div>
```

Comment:
I think a rebase will remove this old stuff here - and after that this should be good to go.